### DISCUSSION TITLE
Loot Tracker Update

### DISCUSSION THREAD

**Discussion by Tradisrad1** [runelite/runelite#17308] [Ideas]
Is it possible to code it so that it filters your bank items similar to how the inventory setups does? I think it would be a great QOL improvement to have your loot populate in your bank instead of having to make a bank tag or search for each item.

**Comment by Firebarrage42** [comment]
I was just thinking of something similar. A plain old search bar to look for items or search by mob name would be nice. I was thinking about playing around with this as a fun first project.
